maps/away/generic/space_bar.dm

```diff
@@ -20,5 +21,9 @@
 	icon_state = "bar"
 	flags = RAD_SHIELDED
 	requires_power = FALSE
-	base_turf = /turf/space
-	no_light_control = TRUE
\ No newline at end of file
+	base_turf = /turf/simulated/floor/plating
+	no_light_control = TRUE
+
+/area/space_bar/hangar
+	name = "Spacer Bar Hangar"
```

Comment:
typo.

Reply:
No, the area for the space bar is called the Spacer Bar, so that is consistent.